jupyter-pytorch-cuda-full/rockcraft.yaml

```diff
@@ -19,8 +19,9 @@ services:
     environment:
        NB_USER: jovyan
        NB_UID: 1001
-       NB_PREFIX: /
-       HOME: /home/jovyan
+       # the following environment variables need to be present in the Pod environment
```

Comment:
let's slitghly update the language here to say `The following environment variables need to be used from the Pod's environment, and not hardcoded.`

Reply:
`to be used` does not make much sense. Let's change it to `to be specified`.